packages/esm-patient-registration-app/src/routes.json

```diff
@@ -45,6 +45,13 @@
       "online": true,
       "offline": true
     },
+    {
```

Comment:
Ditto... I think this actually is a different PR?

Reply:
This is already in main branch. Something went wrong when merging it here. It's fixed now!